src/keymap.js

```diff
@@ -74,6 +74,11 @@ export function baseKeyMaps(schema) {
     bind('Mod-I', toggleMark(schema.marks.em));
   }
 
+  if (schema.marks.superscript) {
+    bind('Mod-s', toggleMark(schema.marks.superscript));
+    bind('Mod-S', toggleMark(schema.marks.superscript));
```

Comment:
```suggestion
    bind('Mod-s', toggleMark(schema.marks.superscript));
    bind('Mod-S', toggleMark(schema.marks.superscript));
```

Can we make this `CMD Shift +` (Microsoft Word) or `CMD .` (Google Docs) instead. `CMD-S` is a known keyboard shortcut for "Save" action

Reply:
Thanks for pointing this out; Will go with the google docs shortcut.